.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSITIVE INFORMATION ABOUT YOUR
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Repository Mirroring Tool (RMT) is an open-source project and we can make it
+ even better with your help and feedback.
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel. If you are having paid support from SUSE
+     business please go to https://scc.suse.com/support for support.
+   - When posting bugs/feedback here, there is no guarantee that the issue
+     will be handled immediately. Depending on the topic and the availability
+     of the developers, the issue maybe resolved in undefined time.
```

Comment:
I'd get rid of the second sentence (Depending on the topic... undefined time.) It basically repeats the first sentence.